### PR TITLE
Add test for kops with aws-encryption-provider plugin

### PR DESCRIPTION
**Pull request by sethpollack**

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -9718,6 +9718,25 @@
...
   },
+  "ci-kubernetes-e2e-kops-aws-encryption-provider": {
+    "args": [
+      "--cluster=e2e-kops-aws-encryption-provider.test-cncf-aws.k8s.io",
+      "--deployment=kops",
+      "--env=KUBE_SSH_USER=admin",
+      "--env=KOPS_FEATURE_FLAGS=+EnableKMSPlugin",
+      "--extract=ci/latest",
+      "--ginkgo-parallel",
+      "--kops-etcd-version=3.1.10",
```
Comment (oomichi) [inline jobs/config.json]: Do we need to specify the etcd version here?
--kops-etcd-version=3.1.10 is specified at ci-kubernetes-e2e-kops-aws-canary only and that comes from https://github.com/kubernetes/test-infra/commit/10753071a8a44f6e4315a72da0ac3091b3147b4c

File: prow/config.yaml [hunk 2]
```diff
@@ -13058,6 +13058,21 @@ periodics:
       image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
       imagePullPolicy: Always
 
+- interval: 4h
+  agent: kubernetes
+  name: ci-kubernetes-e2e-kops-aws-encryption-provider
+  labels:
+    preset-service-account: true
```
Comment (oomichi) [inline prow/config.yaml]: need to change true to "true" as https://github.com/kubernetes/test-infra/commit/1dd7bf44e4d275f9c4d070257d18d8fca7550e10

File: prow/config.yaml [hunk 3]
```diff
@@ -13058,6 +13058,21 @@ periodics:
...
+    preset-aws-ssh: true
```
Comment (oomichi) [inline prow/config.yaml]: ditto

File: prow/config.yaml [hunk 4]
```diff
@@ -13058,6 +13058,21 @@ periodics:
...
+    preset-aws-credential: true
```
Comment (oomichi) [inline prow/config.yaml]: ditto

File: jobs/config.json (hunk 1 above)
Comment (sethpollack) [reply jobs/config.json]: Yes, etcd v3 is a requirement for this test.

**Comment by BenTheElder** [comment]
/sig aws
/cc @justinsb 
@justinsb if not you, can you point me to who should review this from kops/aws side?

**Comment by fejta** [comment]
Looks like this is ready for another round of reviews? (Seth, consider adding a Done comment or something like that when you address previous feedback)

/assign @oomichi 
@kubernetes/sig-aws-misc

**Comment by sethpollack** [comment]
will do

**Pull request closed without merging**